tests/unit/drivers/test_chunk2doc_rank_drivers.py

```diff
@@ -42,6 +48,31 @@ def create_document_to_score():
     return doc
 
 
+def create_chunk_matches_to_score():
+    # doc: (id: 100, level_depth=0)
+    # |- chunks: (id: 10)
+    # |  |- matches: (id: 11, parent_id: 1, score.value: 2, level_depth=1),
+    # |  |- matches: (id: 12, parent_id: 1, score.value: 3, level_depth=1),
+    # |- chunks: (id: 20)
+    #    |- matches: (id: 21, parent_id: 2, score.value: 4, level_depth=1),
+    #    |- matches: (id: 22, parent_id: 2, score.value: 5, level_depth=1)
+    doc = jina_pb2.Document()
+    doc.id = 100
+    num_matches = 2
+    for parent_id in range(1, 3):
+        chunk = doc.chunks.add()
+        chunk.id = parent_id * 10
```

Comment:
can we explicitly set:
`doc.level_depth = 0`
and then:
`chunk.level_depth = doc.level_depth + 1`
and then:
`match.level_depth = chunk.level_depth`

I think it is more explicit and it would be good to keep this examples as good documentation

Reply:
Good point!